inbm/dispatcher-agent/dispatcher/schedule/rpc_activate_operation.py

```diff
@@ -0,0 +1,31 @@
+"""
+    Executes the AMT Rpc Activation command.
+
+    Copyright (C) 2024 Intel Corporation
+    SPDX-License-Identifier: Apache-2.0
+"""
+from typing import Optional
+from inbm_common_lib.shell_runner import PseudoShellRunner
+
+
+
+class RpcActivateOperation: 
+    def __init__(self, url: str, profile_name: str) -> str:
+        """RpcActivateOperation class is to execute rpc activate command
+
+        @param url: url address of the RPS
+        @param profile_name: profile_name used for rpc configuration
+        """
+        pass
+
+    def execute_rpc_activation_cmd(self, url, name) -> str:
+        """Executes the RPC activation command.
+
+        @return: return 'success' if execution succeeds else returns 'Failure'
+        """
+        command = f"./rpc activate -u {url}/activate -n --profile {name}"
+        (out, err, code) = PseudoShellRunner().run(command)
```

Comment:
I was thinking we may want a timeout on this command, but our convention seems to be that we don't do that when running commands, so maybe not necessary here

Reply:
Yes for now I'd keep it like this. In the next story if necessary I'll add to it.